Formula/sonobuoy.rb

```diff
@@ -23,10 +23,7 @@ def install
     (buildpath/"src/github.com/heptio/sonobuoy").install buildpath.children
 
     cd "src/github.com/heptio/sonobuoy" do
-      system "go", "build", "-o", bin/"sonobuoy", "-installsuffix", "static",
-                   "-ldflags",
-                   "-s -w -X github.com/heptio/sonobuoy/pkg/buildinfo.Version=#{version}",
-                   "./"
+      system "make", "native"
```

Comment:
Unless they're shipping their git repo in the tarbal this won't work. You'll need to use a git checkout as source or request that users can pass the version to the makefile as an argument.